uaclient/cli.py

```diff
@@ -511,7 +511,14 @@ def wrapper(*args, **kwargs):
             sys.exit(1)
         except util.UrlError as exc:
             with util.disable_log_to_console():
-                logging.exception(exc)
+                msg = ua_status.MESSAGE_CONNECTIVITY_ERROR
+                if exc.url:
+                    msg += ". Failed to access url: {url}. {error}".format(
+                        url=exc.url, error=exc
+                    )
+                else:
+                    msg += ". {error}".format(error=exc)
```

Comment:
Should we have these as two different templates in `status.py` instead of concatenating?

Reply:
Sure, let's do it.